Program.cs

```diff
@@ -40,12 +40,40 @@
                         }
                     };
                 });
+builder.Services.AddAuthorization(options =>
+{
+    options.AddPolicy("IDSCReferenceTokenView", policy =>
+        policy.RequireAssertion(context  =>
+            context.User.HasClaim(c =>
+                c.Type is "identity-server-config" or "identity-server-config.reference-token" or "identity-server-config.reference-token:view")));
+    
+    options.AddPolicy("IDSCReferenceTokenRevoke", policy =>
+        policy.RequireAssertion(context  =>
+            context.User.HasClaim(c =>
+                c.Type is "identity-server-config" or "identity-server-config.reference-token" or "identity-server-config.reference-token:revoke")));
+    
+    options.AddPolicy("IDSCClientSecretCreate", policy =>
+        policy.RequireAssertion(context => 
+            context.User.HasClaim(c => 
+                c.Type is "identity-server-config" or "identity-server-config.client-secret" or "identity-server-config.client-secret:create")));
+
+    options.AddPolicy("IDSCClientSecretDelete", policy =>
+        policy.RequireAssertion(context =>
+            context.User.HasClaim(c =>
+                c.Type is "identity-server-config" or "identity-server-config.client-secret" or "identity-server-config.client-secret:delete")));
+});
 builder.Services.AddConfigurationDbContext(options =>
                 {
                     options.DefaultSchema = builder.Configuration.GetValue("ConfigurationSchema", "Configuration");
                     options.ConfigureDbContext = b =>
                         b.UseSqlServer(builder.Configuration.GetConnectionString("Configuration"));
                 });
+builder.Services.AddOperationalDbContext(options =>
```

Comment:
We're making the assumption here that both the Configuration store and the Operational store are stored in the same database. Although that is the case for us, it might not be the case for everyone. Perhaps we want to use a separate connection string name (ie. "Operational") so that folks can configure a different one if needed. We'll just put in the same connection string for both.

Reply:
Great point, didn't tink of that one.

Reply:
Fixed in commit da42bfc